extensions/wikia/Search/classes/IndexService/DefaultContent.php

```diff
@@ -83,7 +83,8 @@ public function execute() {
 				'page_images'                => count( $response['parse']['images'] ),
 				'iscontent'                  => $service->isPageIdContent( $pageId ) ? 'true' : 'false',
 				'is_main_page'               => $service->isPageIdMainPage( $pageId ) ? 'true' : 'false',
-				];
+				'indexed'                    => gmdate("Y-m-d\TH:i:s\Z")
```

Comment:
The tests for this extensions are broken heavily and without a big refactoring they will not be usable. This is why I'd rather not adjust them for this change.